### ISSUE TITLE
How to get the input files

### ISSUE THREAD

**Issue by YeehanXiao** [zcang/SpaOTsc#6]
hi,
A great algorithm but i am really confused by the part of processing data when testing it....How could I get the input files from my own scRNA & spatial data? e.g. df_sc_bin, df_is_bin, df_lasso, is_dmat, issc, sc_data_bin...Maybe we need a general pipeline when applying it.
Thanks!

**Comment by tctaochen** [comment]
I have the same question!
I try some method to get these input files.
df_sc_bin/sc_data_bin

`a <- SCTransform(a, assay = "RNA",new.assay.name = 'RNA_normal', verbose = FALSE)
c=a@assays$RNA_normal@data
c[c<0.23]<-0
c[c>0.23]<-1`

is_dmat : I try use R packages 'geodist' 
mcc: I try use R packages 'mltools'

What I can do is just keep the consistency between matrix sizes,without make sure the values which in these matrix meaning is correct.

In conclude,I can't get a reliable result.
Maybe we need more details to get the input files.

Thanks!

**Comment by lidongyu16** [comment]
right here.... 
https://github.com/zcang/SpaOTsc/tree/master/short_tutorial/data

**Comment by zcang** [comment]
Sorry for missing this issue... the *_bin files are the optional binarized expression to compute mcc which gives us the coupling "cost". In practice, we have found that simply using spearman's r on the unbinarized expression works well too. 
We have developed a new package (hopefully much easier to use than this one) for CCC inference (https://github.com/zcang/COMMOT) in ST data with improved algorithms where I have also implemented this integration tool (https://commot.readthedocs.io/en/latest/api/commot.pp.infer_spatial_information.html#commot.pp.infer_spatial_information).
